envs/samtools/samtools-1.9.yaml

```diff
@@ -1,28 +1,37 @@
 name: test-samtools
 channels:
-  - conda-forge
   - bioconda
+  - conda-forge
   - defaults
 dependencies:
   - _libgcc_mutex=0.1
   - _openmp_mutex=4.5
+  - bcftools=1.8
   - bzip2=1.0.8
-  - ca-certificates=2020.4.5.1
+  - ca-certificates=2020.12.8
+  - certifi=2020.12.5
   - curl=7.69.1
   - htslib=1.9
   - krb5=1.17.1
   - libcurl=7.69.1
   - libdeflate=1.5
   - libedit=3.1.20170329
+  - libffi=3.2.1
   - libgcc-ng=9.2.0
   - libgomp=9.2.0
   - libssh2=1.8.2
   - libstdcxx-ng=9.2.0
+  - lofreq=2.1.3.1
```

Comment:
Is the inclusion of lofreq in samtools environment intentional? I thought lofreq has its own environment, where samtools is included, but not the other way around.

Reply:
Does not look like this comment was addressed, can you confirm you looked at it?